js/plugins/omniture.js

```diff
@@ -647,10 +652,14 @@ var OoyalaPlayerDelegate = function()
     videoInfo.id = id;
     videoInfo.name = name;
     videoInfo.length = length;
-    //TODO: StreamType and update unit test
-    //The type of the video asset, one of the following: AssetType.ASSET_TYPE_LIVE,
-    //AssetType.ASSET_TYPE_LINEAR, AssetType.ASSET_TYPE_VOD
-    videoInfo.streamType = ADB.va.plugins.videoplayer.AssetType.ASSET_TYPE_VOD;
+    if (streamType === OO.Analytics.STREAM_TYPE.LIVE_STREAM) {
+      videoInfo.streamType = ADB.va.plugins.videoplayer.AssetType.ASSET_TYPE_LIVE;
+    } else if (streamType === OO.Analytics.STREAM_TYPE.VOD) {
+      videoInfo.streamType = ADB.va.plugins.videoplayer.AssetType.ASSET_TYPE_VOD;
+    }else{
```

Comment:
Minor issue - spacing. Only fix if there are other things to fix.

Reply:
Done the suggested style changes